src/common/Card.tsx

```diff
@@ -38,7 +39,7 @@ const Card: React.FunctionComponent<CardProps> = ({
   if (image) updatedImage = addImageTransformation(image);
 
   return (
-    <div className="card-demo pb-1">
+    <div className="card-demo-pb-1">
```

Comment:
See comment on `index.css`

Reply:
I did add the "-" in the classname making it card-demo-pb-1 instead of card-demo pb-1. The reason I did this as when you don't add the css the paragraph is not align correctly on the page. I will remove and put it back and send you a screenshot of what it looks like without any css. It doesn't display correctly.